pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller_test.go

```diff
@@ -152,36 +111,14 @@ var _ = Describe("NodeNetworkConfigurationPolicy controller predicates", func()
 					"label1": "foo",
 					"label2": "bar",
 				},
-				GenerationOld:   1,
-				GenerationNew:   2,
-				ReconcileCreate: false,
-				ReconcileUpdate: false,
+				Matches: false,
 			}),
 		Entry("node not found",
-			predicateCase{
-				ObjNodeName:     "node02",
-				NodeLabels:      map[string]string{},
-				NodeSelector:    map[string]string{},
-				GenerationOld:   1,
-				GenerationNew:   2,
-				ReconcileCreate: false,
-				ReconcileUpdate: false,
-			}),
-		Entry("generation remains the same",
-			predicateCase{
-				ObjNodeName: "node01",
-				NodeLabels: map[string]string{
-					"label1": "foo",
-					"label2": "bar",
-				},
-				NodeSelector: map[string]string{
-					"label1": "foo",
-					"label2": "bar",
-				},
-				GenerationOld:   1,
```

Comment:
We still have generation check in the predicate, could you keep the test?

Reply:
ack.